lib/Backend/arm64/LowerMD.cpp

```diff
@@ -1645,98 +1646,104 @@ LowererMD::LowerExitInstr(IR::ExitInstr * exitInstr)
             exitInstr);
     }
 
-    int32 spOffset = 0;
+    int32 initialSpOffset = MachRegInt * (usedRegs.Count() + savedDoubleRegs.Count() + (hasCalls ? 2 : 0));
```

Comment:
>MachRegInt [](start = 28, length = 10)

nit: Though MachRegInt and MachRegDouble are same in ARM64 it will be good to use RegInt with usedRegs (+fp,lr) and RegDouble for savedDoubleRegs